_includes/comparison.html

```diff
@@ -226,7 +226,7 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
       <tr>
         <th scope="row">Cloud Integrations</th>
         <td><a href="https://cloud.google.com/istio" target="_blank">Google Cloud</a>, <a href="https://www.alibabacloud.com/help/doc-detail/89805.htm" target="_blank">Alibaba Cloud</a>, <a href="https://cloud.ibm.com/docs/containers?topic=containers-istio" target="_blank">IBM Cloud</a></td>
-        <td><a href="https://marketplace.digitalocean.com/apps/linkerd" target="_blank">DigitalOcean</a></td>
+        <td><a href="https://marketplace.digitalocean.com/apps/linkerd-beta" target="_blank">DigitalOcean</a></td>
```

Comment:
Hello,
can you tell me if there is more meaning behind this change in the given URL to `linkerd-beta` ? I would like to know more about this, but can not find information.

You can contact me via mail (marco.steinke@innoq.com)

Reply:
When I looked at the Digital Ocean page, the `linkerd-beta` page works, and without the `-beta` it's a broken link. 🤷‍♂️